docs/source/conf.py

```diff
@@ -93,13 +95,22 @@ def is_stable(extra):
     tags.add('ipystable')
 else:
     tags.add('ipydev')
-    rst_prolog = """
-    .. warning::
+    rst_prolog += """
+.. warning::
+
+    This documentation is for a development version of IPython. There may be
+    significant differences from the latest stable release.
+"""
+
+rst_prolog += """
+.. important::
 
-        This documentation is for a development version of IPython. There may be
-        significant differences from the latest stable release.
+    This is the documentation for IPython version > 6.0 which is had
+    stopped compatibility for python version lower than 3.3. If you are
+    looking for a version of IPython compatible with python 2.7 please see
+    the documentation for the IPython 5.x LTS (Long term support branch)
 
```

Comment:
This documentation covers IPython versions 6.0 and higher. Beginning with version 6.0, IPython stopped supporting compatibility with Python versions lower than 3.3 including all versions of Python 2.7.

If you are looking for an IPython version compatible with Python 2.7, please use the IPython 5.x LTS release and refer to its documentation (LTS is the long term support release).